docs/EN_US/DynamicESDL/DESDL-Mods/ESDLmax_count_var.xml

```diff
@@ -6,13 +6,17 @@
 
   <para><indexterm>
       <primary>max_count_var</primary>
-    </indexterm>The max_count_var attribute is used to specify the expected
-  max items in a dataset (ESParray).</para>
+    </indexterm>The max_count_var attribute is used to specify a variable (an
+  ECL Attribute) cotaining the value of the expected max items in a dataset
+  (ESParray).</para>
 
   <para><emphasis role="bold">Example:</emphasis></para>
 
   <programlisting>[max_count_var("iesp.Constants.JD.MaxRecords")] ESParray &lt;ESPstruct MYRecord, Record&gt; Records;   </programlisting>
 
   <para>Roxie will define the constant iesp.Constants.JD.MaxRecords and change
   it at will without affecting ESP.</para>
```

Comment:
and if you change above, then instead of "without affecting ESP", use "rather than hard coding the value in the ESDL"?